peopleops/index.md

```diff
@@ -0,0 +1,17 @@
+## Vacation Policy
+
+FlowForge has a unlimited time off policy. Taking vacation is encouraged for all
+team members. To prevent an undefined number of vacation days to start a race to
+the bottom, we recommend each team member to take a minimum of 25 days a year,
+and at least 5 days a quarter.
+
+### Logging time off
+
+When taking more than 3 days off consecutively, tell (don't ask) your manager. 
+This ensure scheduling of work and operations continue to run smoothly without
+you. When taking over 2 weeks of consecutively seek approval from your manager.
+
+Before you can take time off you should _always_:
+* Log your time off in Pilot.co
```

Comment:
@sammachin Could you validate Pilot allows us to do this? I couldn't find it in the interface just now, though I'm legally a contractor.

Reply:
Checked with Pilot support today, Pilot doesn't record PTO. BambooHR does.

Reply:
BambooHR (which how we interact with pilot) has a "PTO Approver" who I assume is someone at Pilot, I wonder if we can have another person added as a 'flowforge manager'

Reply:
Put in Bamboo instead of Pilot. Thanks for figuring this out @sammachin

Reply:
Sorry yes my mixup, although as an employee Bamboo Is all I really see of pilot it has, holiday, payslips, contracts etc. The actual [PilotHR](https://work.pilot.co/) webapp has nothing really in it other than expenses (which we have Brex for) So I think of Bamboo as Pilot